doc/api_methods.md

```diff
@@ -70,14 +70,14 @@ Returns an array of layers in the map. The first is the base layer.
 
 ### vis.addOverlay(_options_)
 
-Adds an overlay to the map that can be either a zoom control, a tooltip or an infobox.
+Adds an overlay to the map that can be either a zoom control, a tooltip or an infowindow (hover action).
 
```

Comment:
`infofox` and `infowindow` are not the same thing. There's an example of an inbox in [this example](https://github.com/CartoDB/cartodb.js/tree/develop/examples/TheHobbitLocations) (I hope it still works).